creating-campaign/stripe-payment-feature/stripe.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Stripe Connected Payments'
+---
+
+<Note>To enable the Stripe feature and connect your account to Moongate's Stripe account, please contact Moongate for permission.</Note>
+
+## What is Stripe Connect?
+> Stripe Connect is a service that allows platforms to manage payments on behalf of their users. Here's an example to illustrate how it works:
+>
+> Imagine you are an event organizer using Moongate to host and manage your events. You want to sell tickets online and receive payments from attendees. Instead of setting up your own payment processing system, you can use Stripe Connect. Moongate, as the platform, handles the payment processing for you. This means:
```

Comment:
I dont think this is correct, Moongate does not handle payment processing. Moongate is a ticketing platform that uses Stripe as a gateway. Organizers never setup their own payment processing system?